Utils/Types/test/ut/CircularBuffer/Main.cpp

```diff
@@ -172,7 +172,22 @@ TEST(CircularBufferTests, BasicRotateBadTest) {
     rotateBad.apply(state);
 }
 
+/**
+ * Test boundary cases
+ */
+TEST(CircularBufferTests, BoundaryCases) {
+    MockTypes::CircularState state;
+    // Serialize an empty buffer
+    state.setRandom(0, 0, 0);
+    Types::SerializeOkRule serializeOk("serializeOk");
+    serializeOk.apply(state);
+    // Serialize a max size buffer
+    state.setRandom(MAX_BUFFER_SIZE, 0, 0);
+    serializeOk.apply(state);
+}
+
 int main(int argc, char* argv[]) {
     ::testing::InitGoogleTest(&argc, argv);
+    STest::Random::seed();
```

Comment:
Without this statement was it always running the same cases?  If so, we should ensure that is called in the other rules-based testing.

Reply:
Yes, we should call it. I believe if you don't seed the pseudo-random number generator, then you get a default seed.

Reply:
Also, there's a way to capture and repeat a particular seed, but it requires going into the CMake build cache to execute the test. Ideally we should add this capability to the fprime-util interface.